homeassistant/components/light/lifx.py

```diff
@@ -0,0 +1,95 @@
+"""
+homeassistant.components.light.lifx
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+HA platform implementing LIFX lights.
+
+"""
+
+import logging
+import lazylights
```

Comment:
This should be imported inside your methods that use it or else Home Assistant can't include this file and install lazylights on demand.